docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -195,4 +195,16 @@ Alternatively, navigate to your Data storage directory and enter the following c
 
 *Further Reading*: https://docs.couchbase.com/server/current/learn/data/data.html#metadata[Metadata], https://docs.couchbase.com/server/current/rest-api/rest-bucket-create.html#evictionpolicy[evictionPolicy].
 
+[#CB90061]
+=== Disk Write Queue (CB90061)
+
+*Background*: Having items in the disk write queue for too long could cause backoffs. This usually indicates either a sizing issue, or a storage medium failing.
+
+*Condition*: Age of items in disk write queue over 50 seconds produces a warning, and upgraded to critical if the age is over 100 seconds.
```

Comment:
Might be worth adding a little more detail about what "age of items" means? Because it's not immediately obvious (it confused a couple of us so it may well confuse users)

Reply:
Changed it to say "If the time in seconds since the most recently stored object was initially queued is over 50 seconds a warning is produced, and upgraded to critical if the time is over 100 seconds."

Reply:
Wait I'll make that better "If the time since the most recently stored object was initially queued is over 50 seconds a warning is produced, and upgraded to critical if the time is over 100 seconds."

Reply:
That's really good - just maybe make the second part its own sentence because it's a bit long?

Reply:
How about "If the time since the most recently stored object was initially queued is over 50 seconds a warning is produced. If it is over 100 a critical alert is produced."?

Reply:
Yep - the first sentence is still a bit long but I'm not sure there's much more we can do there.

Reply:
could be worth clarifying "most recently stored" as most recently persisted to disk. And queued as added to Disk Write Queue.

Reply:
I've changed it to "If the time since the object most recently persisted to disk was added to the Disk Write Queue is over 50 seconds, a warning is produced. If that time breaches 100 seconds, then a critical alert is produced." how is that?